src/targeting/Semgrepignore.mli

```diff
@@ -33,6 +33,7 @@ type exclusion_mechanism = {
 *)
 val create :
   ?cli_patterns:string list ->
+  ?semgrepignore_filename:string option ->
```

Comment:
the type should be `string`.

after you change it the compiler will help you adapt other places as required.